releases/release-1.31/release-notes/maps/pr-124316-map.yaml

```
@@ -0,0 +1,4 @@
+pr: 124316
+releasenote:
+  text: 'ACTION-REQUIRED: DRA drivers using the v1alpha2 kubelet gRPC API are no longer
```

```suggestion
  text: 'ACTION-REQUIRED: Dynamic Resource Allocation (DRA) drivers using the v1alpha2 kubelet gRPC API are no longer
```

https://github.com/kubernetes/sig-release/pull/2542#discussion_r1653206198